plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/glue/TestDeltaLakeGlueMetastore.java

```diff
@@ -315,4 +332,38 @@ private static String randomName()
     {
         return UUID.randomUUID().toString().toLowerCase(ENGLISH).replace("-", "");
     }
+
+    private void cleanupOrphanedDatabases()
```

Comment:
Can we put this in a separate test Class? It could be nice to run this class as a suite without needing to go through and clean all orphan databases

Reply:
@alexjo2144 please tell me if this is what you wanted

Reply:
It is, thanks